### PR TITLE
adds rare tier to maint loot

### PR DESCRIPTION
**Pull request by Fikou**
## About The Pull Request

adds rare tier to maint loot, somewhere between oddities and uncommon items, you should probably encounter 1-2 per round
moves some stuff from uncommon and oddity into this, notably the pipegun recipe and yellow gloves/justice helmet
adds some unused items into these pools, like the flashdark, paint (rarity) and cloaker belt, grey tide spear (oddity)

## Why It's Good For The Game

oddities comment says "strange or crazy items"
insulated gloves or a justice helmet arent crazy in any form
some more variety is always nice

## Changelog
:cl:
balance: adds rare tier to maint loot, expect to sometimes find some neat stuff!
/:cl:

### CODE REVIEW

File: code/_globalvars/lists/maintenance_loot.dm [hunk 1]
```diff
@@ -240,13 +243,71 @@ GLOBAL_LIST_INIT(uncommon_loot, list(//uncommon: useful items
...
+	list(//tools
+		/obj/item/weldingtool/hugetank = 1,
+		/obj/item/kitchen/knife = 1,
+		/obj/item/restraints/handcuffs = 1,
+		/obj/item/shield/riot/buckler = 1,
+		/obj/item/melee/baton/cattleprod/teleprod = 1,
+		/obj/item/throwing_star = 1,
+		/obj/item/pen/survival = 1,
+		/obj/item/flashlight/flashdark = 1,
+		/obj/item/assembly/flash/memorizer = 1,
```
Comment (necromanceranne) [inline code/_globalvars/lists/maintenance_loot.dm:258]: So I went and looked up what this is. It's just a flash with a different sprite. How sad!

**Comment by kopoba** [comment]
good good sunglasses are back

File: code/_globalvars/lists/maintenance_loot.dm (hunk 1 above)
Comment (Fikou) [reply code/_globalvars/lists/maintenance_loot.dm:258]: yeah its for extreme roleplay you wouldnt understand

Comment (necromanceranne) [reply code/_globalvars/lists/maintenance_loot.dm:258]: Don't worry I'll just use the hypnoflash for my extreme roleplay ;)

**Comment by Fikou** [comment]
image
went through delta maint as an assistant, not an exact science but i found one rare item so it seems to work right

File: code/_globalvars/lists/maintenance_loot.dm (hunk 1 above)
Comment (interestingusernam3) [reply code/_globalvars/lists/maintenance_loot.dm:258]: Manuel RP consumers vs Citadel ‏ ̴̡̧̢̡̡̨̧̡̨̧̛̛̛̺̯̘̪̠̬̝͍̭̼̬̣͓͇̼̺̩͇̹̫̞̗͓̤̞̫͖̬̠̗̮̭͕͚͍̯̖̥͉͕͓̦̟͙̜̳̥͙̹͈͎͔͉̳̖̩͕̬̝̩̻̮̝̝̙̲̣̪͍͖̗͚̹̖͖̼̖͍̤͉͓̞͚̗̉̀̐̏̄̅̽͌̉̌̓͆̄͐̀̓̌́̈́͌̐̆͛̿͒͋̍̉͒̍̒̆̅̇̽̋̉̑̿́͛̔̎̄͊̀͂̆̈́͆̓̈̈́͒̀̅̓̀͗́̿͆̇̐̽̌́̒̋̄̍̈́́̍̄̓̇͑̌̉́̓̽̉̐̕͘̕̕̚̕͜͜͝͝͠͝͝͠͠͝ͅͅت̵̨̨̡̧͖̗̲͓̟̯̪͉͔͕̘̦͈̰̻͖̫̲̪̠͉̫̲̲̞̼̗͙͉̤̟̠̩͖̘̟͕̣͚̙̣͓̩͈͔̮̱̖͎̺̯̦͙̰̳͎̤̮̹̭̦̤̙̰̗͎͉̟̳̺̝̖̳̰̩̹̦̯̝̞͈̼̽͋̉͛̾͆̌͌̐̄̚̚͜͠ͅͅخ̵̨̨̧̡̨̨̧̡̛̛̛̛̛̞̩͍̬͙̥̗̘̮̤͎̬͙̲͕͙̞͍̳̥͎̣̩̳͍̟͚̩̰̘̭͖͙̪͕̪̖͔̹͔̱̗͙̹͈̤͈̘͚̼̙͕̞͈̭̠̲̱̝̠̘̗̘̭̻̥̞͚̎̔̾̌̐̂͛̀̉͗̈́̽͑̈́̀̍̈́̂͐̀̀̽̄̐̽͊͗̈́͗̇̿̒̈́͑̿̊̒̈́̽̌̽̈́͑̊̓̃͂̑̃̂́͆͋͂͑͊̚̚̚͘̕͜͜͠͠ͅط̴̨̢̨̨̡̧̡̡̛̘̟̫͔̣͈̲͚̩͎̭̯͔̟̠̮̟̺̼̫̦̘̱̤͎͍̺͔̩͎̪̪̫͙̙̝̪̗̦̦̙̬̼͍͕͖̰̙̲̩̺̞̻̺̭̤̯̩͔̟̙̳̦̗̫̱̖͉̻̦̦̪͍̻͈͇̗̜͙̫̲̱̲̪͎̘̱̠͔̤̞͔͔͉̬̣̬̲̹̣̥͇̙̝̍͆͋͐̎̉͐́͐̀͂̒͌͋̆͌̌̏̀̈́̔̋̑̐̃̆̎͐̈́̔͗̽͊͌̉͘̚̕͘͜͜͜͝͝͝͝ي̶̢̛̛̻̲̣͚͇̰̥̳͕͕͍͙̉̆̅̋̀͒̊̓́̿̈́̈̊͗̾̋̇̓̄̆̓̾͛̿̀̊̃̇̋͂͆̔́̽̆̍́͌̅͛̈́͋̏̎̈́̀͆̃̓͐̑́̾̍̐̑́̾̎̽̅̀̋̿̅̔̑̿͛͐͆̄̓͆͌̔̓͌͑͊̉͗͒̀͂̏̒̐́̀̏͛̓̐͋͋͘͘̚̕̚͜͜͠͝͝͠͠͝͠͠ͅط̵̧̛̛̻̳̤̟̥̻̖͕͉̬̰̣̤̥̱͓͈̥̳̩̠͇̩̼̞͉̭̮͉̥͍̭̼̔͑̓̊̓̀̒͋́͆̍̏̐̏͌̾̿͑̄̔̔̎̂̏̀̃̽̓͗͑̇̆́̍͋̃̎̉́̈̌͋̅̉̀̽̔͑̒̌̈̇͛̌͗̅͋͗͛͒͂͂̃͒̌̓̅̀̿͋̐̌̈͗̽̆̓͐͑͗̽̅̈́̃̇͛͌̀̏͂̉̑̾͐̓̋̓̉̀͛̃̈̂͊͊͋̄̄͒̎̽͐̆͐̉́̽̀̏̈͒́́͛̀̔̑̓̉̈́̍̓̇͑̑̎͌́͗̏̒͂̋̕̚͘͘͘̚͘̚̚͜͜͠͝͠͠͠͠͝͝͝͝͝ͅ ̷̨̛̛̛̛̛̛̛̛̛̞̳̱͈͕̜̣̩͔̩̲̈́̒̅͐̄̊̔̈̏̿̀͊̈́̂̐̋̈́͊̆̃̇̆̊̐́͛͌̀̔̑͗̒̓͌͋̐̂̋̅̓̇̆̑͑͒̄̑̀͌̀̿̈́̍̔͆̿̿̅̇͒̆̓͌͆̈́̊͒̓̃͒̾̈̂́̊͑̈́͆̈́́̀̿̈̊̉͆͆̈́̀̆̄̎̽͑̽̾͗́͗̽̎̑̍͌̏̈̋̾̍̂̿̓͛̈́̒̓̇̔͊̐̏͛͆̓͂͊̒̾̃͂̔́̊̄͐͌́͗̿̾̿͘͘͘̕͘͘̕̕͠͝͝͠͠͝͠͝͠͝͝͝م̴̨̧̡͎̮͓̳̭̞̻͔̖̟͇̠̲͖̲̯̺̦͈̠̹̦̜̞͚͉̩̹͔͙̘̏͆̎̐͋̋̍͜͜͠و̷̨̡̡̡̡̡̨̧̧̧̢̨̡̢̧̧̨̛̼̰̬͚̙̱͉̯̥͕̟̺̣͙̗̻̤̲̺̘̗͇͓͙͍̫̠̠̹̖̲͍̙͓̯̼̲̘̲̝͎̞͔̹̞͖̼̙͎̰͖̞̳̞̜̟̭͈̠̣̤͔̫͖̮͕̟̲̪̤͕͖̳̲͈͕̬̘̞͍̤̤͎̭͙̣̹̦̭͈͙̯̦͎̥̣͚͔̲̣̗̙͉̮̘̣̣̖̥͇͙̬̹̫̖͉̜͙͖̥͎͖̼͇͎͕̻̪̯̳̣̜̝̪͕̭̌̏̀̈́͊̿̅̀̑͗̈́̀̓̿̽̎̋̿̈̏͊͆̓̐̇̽́͂̇̄̈́̅͋͑͑̕̕͜͜͜͜͜͜͝ͅͅͅا̴̨̡̢̡̢̨̡̧̺͙̞̲̱̟̙̘͈͍͈̼̼̖̩̞̲̥̣̯̩͔͉̤̣̺͇̣̣̣͖̰͉̦̯͙̯̳͉̪̙̱̬͓̲͉͉̩̳̝̥̳͉͓̫̩̥̩͙̹̩͔̤̞͔̙̬̠̖̩̭̼̼̠͉͔̺͔̲̯̞̜͉̟̼̫͙͍̤̩͇̫͕͖̜̬̦̫̹͙̠̳̙̹̱͖̮͍͙̑͋͂̄̍̏́̂͋́̏̆͊̋̈́͘͘͜͜͠͝͝ͅͅͅͅر̶̧̢͚̫̲̺͚̣̟͔͔̣̼͇̟̰̠̯͍̠̭͎̟̱̠̩͉̻̟̬͉͚̜̬͉͙̗͓͙̠̩̭͉̹͎͎̟̣̬̹̥̟͔̱̗̤͆̀̉̾̎̆̒̅̌̄͛͌̀̾̓̅̓̐͗͑̐̓̈́́̄͑͊̌̍̀̿̅̅̀͐̌̆̐͑̏̿͐̎͐̆̋̃̈́̉́͒͛̒̓́̍̒͌̐̽̀̀̾̌̍̀̔̄̀͛̍̄̊̑̐͊͆͘̕̕̕̕͘͘͘̕͠͝͠͝͝͝ͅد̵̧̧̢̢̡̧̨̨̢̨̨̢̛̛̜͉̼̯͕̦̠̲̬͉̥̪̹̘͔͎̭̱̹̯̬̪̥͈͖͓̪̻̖͖̜̗̯̪͕̹̜̩͙̖̠̪͚̞͍̳̯̞͎͙͈̝̠͍͎͕̺̫͓̫̼͖̝̦̫̟͖͕͎̙̹̭̘͙̦̯̱̯̩̳͎̫̖͍͉̝͕͔͕͔̤̭̙͕̭̹̜̟̼̹͖̦̟̟͓̬̝͙̲̙͍̱̬̻̣̥̯͚̙͉̪̥̦͔͙̺̱̼͔̺̞̙̤͍̯̝̯̭̩̠̰̟̪̼̂̀͌̉̒̌̈́̄̈́̽̋͊̉́̐̽̈́̊̏̽̈̑̀̍̎̑͑̓̑̔͋̋̓̃̈̈́͗̂́̔͛̓̋̆̍͂̋̍́͐͐́̐͌̓͗͆͆̀̐̊͘͘͜͜͜͝͠͝͝͝͝ͅͅͅͅͅ ̵̨̢̧̼̗̮͍̯͎̞̥̲̣̗̹̜̹͙̘̰͕̣̫̦̬̟̜̣̉̓̈́̉́̈́̒́̑̒̾̓̒̏̍̈̿͆͒̌̃̔̓̂͂͒̔͒̿̇́̍̒̈͛͌̆̈́͂̀̀̍͌̉͌̍̈̊̉͋̓̎̎̒́̕͘̕͘͘͘͜͜͝͝͝͝͝͠ͅا̵̢̧̨̨̨̧̧̡̡̡̨̧̧̡̨̢̡̢̛̛̛̛̛̛̤̟̪̟̲̣̼͎̰͚̪͚̥̤̹̻̳̻͔̙̣̪̥̦̠͕̦̲͓̘̗̭͉͙̺̺̹̜̼̣̖͙̞̖̗̲͉̞̳̙͎͖̬̰̯̯͕̗̲̱̳͍͓͉̰̬̠̱̖̪̮̺̜̯̜̣̠͉͖̯̘̥͍̣͚͇̘̯̙͙̘̬̳̭̣̥̺̮̰̹̦͉̦͈̹̙̻͍̯͉͕̝̯̖͓̦͍͔̯̬͍͓̘̘͈͍̯̞̟͓̰̪̪̱̊͒̀̀̌͒̈́̑̑͐̓̈́̎͂͋͌̃̏̊͌̍̐̌́͋̾̿̾̌́̏͛̒͆̈́̽͂̽̋̊̒͌̌̽̆̈̋͒̂̓́̉̈́̎̋̉͗͌̍̏́͊̾̍̌̾͗̋̈̃̾͛̈̒͐̒̈́̆̏͛̈́́̿̃̈́͆̄̿̒͘͘͘̚̕͘͜͝͝͝͝ͅͅل̷̢̡̧̧̧̡̨̛̪̩͈͓̬̳̟̲͔̫͇̘̭̳̻͓̭̣̘͕̣͔̮͈̖͙̺͖̹̜͚͚̣̗͎̱͔͍̪̠̰̫̠̩̱̪͇̳̪̝̞͔̼͓̙͎̻͇͕̞͖̗̖̰͕̗͚͔̗̦͈̹͓̹̫̩̣̫̯̃̓͗̋́̓̉̀͂̈̎̽͒̍̆̓͊́̋͑̃̌͋̿͐͑̈́̇̆͊̐̌́̇̈́́̋̒̂͊́̀̂̃̔̈̽̎̐̆̒͂̈́̈͂̽̏̅̀̔̆̄̑̐̿̀͒̃̔̾̍̈̏̍̑͆́̊̉͌̓̐͗̀̀̂͒̈̉̈́͒̈́̈̿̐̓̃̐̿͑̅̚͘̚̕͜͜͠͠͠͝͝͝͝͝͠͠ͅͅͅم̸̧̡̢̛̻͇̺̝̺̣͔̗̱͕̬̯͚̹̗̖̹̬̰͈͎̟͉̥̹͔͈͙̭̦̻̳͖̟͍̭̭̏̆͗̈̋̈̓̌̊̏̔͑͋̊̿͌͛͂̾͌̅͊̋̈́͛̓̃͐̀͂̿̄͊̌̎̉̋́̑́͐̐͋́͊̌̂̈́͗̇̆͂̄̎͊̄͛̄̈̉̓͌͌̐̌̄̏͒̊̈́̽̍̆́͐̅͒̐̊̊́͌͌̾̊̂̈̅̄͌͆͑͑̂̋̍̚͘̚͘͜͜͝͝͝͝͠͠͠͝͠͝ش̸̢̨̛̛̛̛͙̹̣̼̰̳̠̲̗̞͍̹̟̳̱̺͚̙͂͒͑̇͊͋̽̈̎̽͗̈́̊͑͆̏̔̋͆̑̄͆̈́̈́̔̈́̀̿̉̋̍͊̿̆̂̅̏̇̄̃̒̍͛̃̓́́̑͆̾̄̈́̒̇̄͂̓͆͛̐͊̽͑̇̿̿̇̒̀͋̋́̑̾́́̇̽̓͒͊̌̅͂̀̓͒͌͒͂̈́͂̃̈̽͊̓̈́͗͛̎̒̍̓́͌́̊̿̀̃̈́͐͑͘̚̕̚͘̕̕͝͠͝͝͠͝͝͝͝͝͠͠ا̵̧̡̢̢̢̡̛̛͍̗̭̹̗͔̲̗͍̗͉̠̱͕͖̘̜̻̬̖̣͇̥͙̰̥͎̼̹͉̦̹̺̩͍̟̻͈̱̠̬͖͎͔̠̩̱̮͎̞̣͈̙̯͍̬͙̟̬̟͙͉̞̥̤̻͉͔͙̝̬̲͓̳̟̣̩̫̘̲̫͂̐̔̈́͛̓̓̈̈̍̈̍͑̔̽̅͗́̍͑̀͒̓̈́̀̐̆̿̓͋́̄̒͗̅̋̈́͌̑̽̐̀͒̈́͋̆̂͊͒͒͛̃͒̈́͑̋̔͑̏͆̋̊͐̀͆̒̎̈̀̐̄̒͑̔̂̾́̇̾̊͌̍̈͋͆̐̔̊̈́̎̀̑͒̅̒̃̋͋̇́͌̀̓̔̒̂́̍̅̾̓̓̈́̀̌̏̅̍͋̾͌̉͋̂͋̈́̋̌̌̂̂͒̽̆̏́̍̕̚̕͘̕̚̕͘̕͘͜͜͝͝͝͝͠͝͝͠ͅر̴̢̢̧̨̢̨̨̢̡̧̢̛̛̛̛̛̛̛̛͍̤̭͍̹̳͇͔̻̯̣̭͍̘̠̭̬͇͎̥͔͖͎̼̠̣̩̩̟̭͕͉̺̪̦̼̞̦̥̗̩͕̲̗̥̯̞̬̼̜͓̬̺͈̙͙͚̜̤̼̰̩̘̺̭̖̥̻̙̻̺͉̻̙̖͚̖̥͓̫̟͉̪̪͎̲̩̝̳̘̙̯̫͓̜͕̝̝̼̱̘̟̫̖̯͎̘̼̰̈́͐̏̆̆̀̀̑̑̔́̀̓͒̀͒̄͌̿̌̇͑͌̀̉́͋̄̋̾́͊̒̂͋͌̒̿͐̓́̆̒̐̏̓͆͑͒̆̆̆͌̍̓̌͐̋̋̔͗́̅̔̍͛̋͊̽̊͂͑͗̾̓́̏͑́̾̐̿̅̓̏̀̈͌̿͂̀͆̈́͌̉̀͌̓̑̈́͑̎̿̏́̃̆̃͐̊̔͐̐͐̄͐̊̇̄̏͘͘̕̕̚̕̕͘͜͜͜͠͠͝͝͝͝͠͝͝͝ͅͅي̸̨̢̧̨̡̨̢̨̢̧̧̨̨̧̢̧̨͕̗̖͙̰̖̖̼̦̻̲̣̘͍͇̠̳̱̮̺͈̠̰̗̣̤͎͔̩̺̺͇̥͉̙̺̞̻̗̜̞̗̝͔͚̙͚̙̻͙̪̻̩̖̻̼̖̩̫̜̹̜̻̥̲̹̫̭̼͈͈̠͙̤̪̬͔͚̦̟̲̺͕̟̭̦̘͎̹͚̬̬̺̬̻̝̦͍̝̙̖̠̙̟̥̖̮̰̹̳͚̖̲̻̰̼̥̳͔̣̪̻͓̰̳̪̗͚̪̙̟͍͔̫̫͇͈͖̘̻̪̝̹͖͎̱̭̖̩̤̋͊̃̈́̆͛̏̔͂̓͂̉͌̔͂̈̀̂̓͑̀̒͂̀̈́͛̎͆̍͐̈́̉͆́̿͆̇̅̌̌̀̒̕͘͜͜͝͝͝͝͠ͅͅͅͅͅع̵̨̧̧̡̢̧̧̢̨̧̧̧̧̧̧̨̡̧̡̡̡̢̛̻͉̲͚̠̖̭̞̲͚̗̦̼̞̲̤̜̠̻̥̝̮̪̹̗̜̟͈̣͓͎͖͈̤̘̭̱̘̜͎̼̜̣̯̦̗̮̳̤̖͉̭̝͚͕̙̱͇͖͕͉̲̜̤̹͉̤͍͍̭̺̜͇͔̘̻̭̮̦̩̖̬̭̘̙͚̼̠͇̼̙̝̱͎̝̟̼͈̰̙̩͙̟̱͖͎̞̀͛̅́̀̈́͌̊̈͋̈́̈͊̀̎͗̽̌̄̓̽͋̊́̾̀̊̈̔͘͜͜͜͜͝ͅͅͅͅ enjoyers

File: code/_globalvars/lists/maintenance_loot.dm [hunk 2]
```diff
@@ -240,13 +243,71 @@ GLOBAL_LIST_INIT(uncommon_loot, list(//uncommon: useful items
...
+			) = 1,
+		list(//medical chems
+			/obj/item/reagent_containers/hypospray/medipen/salacid = 1,
+			/obj/item/reagent_containers/hypospray/medipen/oxandrolone = 1,
+			/obj/item/reagent_containers/syringe/contraband/methamphetamine = 1,
+			) = 1,
+		) = 1,
+
+	list(//misc
+		/obj/item/disk/nuclear/fake = 1,
```
Comment (coiax) [inline code/_globalvars/lists/maintenance_loot.dm:298]: Can this be deconned for syndicate tech? But also ❤️

Comment (Fikou) [reply code/_globalvars/lists/maintenance_loot.dm:298]: no i removed that like a year ago i think

**Comment by Tlaltecuhtli** [comment]
add maint rocket launcher and blackmarket uplinks

**Comment by Megarop** [comment]
add maint antag items

**Comment by Pugie** [comment]
I am beginning to believe maint die of fate was a lie and I am gullible add that

**Comment by necromanceranne** [comment]
> I am beginning to believe maint die of fate was a lie and I am gullible add that

It's real man, keep rolling

**Pull request merged**